lib/modules/keyboardNav.js

```diff
@@ -1219,8 +1219,16 @@ function reply(selected = getSelected()) {
 	));
 }
 
-function navigateTo(newWindow, href) {
-	if (newWindow) {
+function navigateTo(href, options: { newWindow?: boolean, leaveNP?: boolean }) {
```

Comment:
`options` can be deconstructed to `{ newWindow = false, leaveNP = false }`

Reply:
Yes, but then each new option we add support for in navigateTo would have to be specified twice - once in the deconstruction, and once in the type definition. If we want that, I'll make the change

Reply:
I think @larsjohnsen is suggesting to combine the deconstruction and type definition by replacing `options: { newWindow?: boolean, leaveNP?: boolean }` entirely with that snippet.

Reply:
Both ways are fine with me.

It would also be worthwhile to make the object type exact (`{| newWindow?: boolean, leaveNP?: boolean |}`) to prevent typos when invoking the function.

Reply:
Replacing ```options: { newWindow?: boolean, leaveNP?: boolean }``` entirely with ```{ newWindow = false, leaveNP = false }``` results in flow errors

**Cannot call navigateTo with object literal bound to the second parameter because property leaveNP is missing in object literal**

for every time navigateTo is called with one of the options missing (like ```navigateTo('/', {newWindow: true})```). I'll try to figure out how to make this work.

Reply:
> Both ways are fine with me.
> 
> It would also be worthwhile to make the object type exact (`{| newWindow?: boolean, leaveNP?: boolean |}`) to prevent typos when invoking the function.

Making function param destructuring work with flow types is a bit harder than I expected. I've made the 'object type exact' change for now.